src/main/java/frc/robot/constants/HardwareConstants.java

```diff
@@ -120,7 +120,8 @@ public record ArmConstants(
             RobotMap.CanivoreCANBus,
             14,
             15,
-            0,
+            //TODO: Do we change all other ids so that this is in order. Or do we just add this one out of order?
```

Comment:
Which IDs come after this? Just 16-18? If it's only those then I think I'd be fine with a reorder.